package_bag/routines.py

```diff
@@ -156,16 +157,9 @@ def create_package(self, bag, bag_json):
         mkdir(package_root)
         with open("{}.json".format(join(package_root, bag.bag_identifier)), "w",) as f:
             json.dump(bag_json, f, indent=4, sort_keys=True, default=str)
-        bag_tar_filename = "{}.tar.gz".format(bag.bag_identifier)
```

Comment:
You actually need to make two tarfiles, one of the bag, and then one that contains the tarfile of the bag as well as the JSON. So you need to call `make_tarfile` twice.